projects/sb-user-management/src/lib/user-count.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
```

Comment:
@poojakarma 
Please move these service files in the user-count folder

Reply:
Resolved